src/main/java/com/microsoft/graph/http/ReferenceRequestBody.java

```diff
@@ -62,36 +52,13 @@ public ReferenceRequestBody(@Nonnull final String payload) {
         odataId = payload;
     }
 
-    /**
-     * Gets the raw representation of this class
-     *
-     * @return the raw representation of this class
-     */
-    @Nullable
-    public JsonObject getRawObject() {
-        return rawObject;
-    }
-
-    /**
-     * Gets serializer
-     *
-     * @return the serializer
-     */
-    @Nullable
-    @Override
-    public ISerializer getSerializer() {
-        return serializer;
-    }
-
     /**
      * Sets the raw JSON object
      *
      * @param serializer the serializer
      * @param json       the JSON object to set this object to
      */
     public void setRawObject(@Nonnull final ISerializer serializer, @Nonnull final JsonObject json) {
```

Comment:
Why does setRawObject remain? Is it overridden else where?

Reply:
I see that it is necessary in case of TypeHelperJava.UpdatePropertiesWithinSetRawObject in typewriter.

Reply:
@MIchaelMainer we do that so the additional data manager gets populated. In some cases the serializer will handle it itself, and then it's a noop. But in other cases, the generation pushes an implementation so I populates addiationalDataManagers of the child elements.

Reply:
Would it make sense to change access to `protected`? Is there a case for a dev to set this?

Reply:
Protected would not give me access in other packages (namespaces ) of the library, or in the service library. https://www.bing.com/images/search?view=detailV2&ccid=vTbYgOGe&id=7B0A47080C48A922C2312EF29407E8EAA6730050&thid=OIP.vTbYgOGeESPalZGV_AYCrQHaET&mediaurl=https%3A%2F%2Fth.bing.com%2Fth%2Fid%2FRbd36d880e19e1123da959195fc0602ad%3Frik%3DUABzpuroB5TyLg&riu=http%3A%2F%2Fcdn.journaldev.com%2Fwp-content%2Fuploads%2F2013%2F10%2Fjava-access-modifiers-table-1.png&ehk=vIsMWMM%2F21QnzOW7NBhOCTkyEqXP%2FKyxf3b0Tk%2FXCOM%3D&risl=&pid=ImgRaw&exph=931&expw=1600&q=java+protected+access+modifier&simid=608011763095831121&ck=A677AD2B5C0E0479FAB1EF627752654F&selectedindex=0&form=EX0023&pc=LNCH10&adlt=demote&shtp=GetUrl&shid=0e769672-debe-460c-92c9-7f43c80d1355&shtk=SmF2YSBBY2Nlc3MgTW9kaWZpZXJzIC0gSm91cm5hbERldg%3D%3D&shdk=Rm91bmQgb24gQmluZyBmcm9tIHd3dy5qb3VybmFsZGV2LmNvbQ%3D%3D&shhk=p08o1TH6%2BPpJM4fzqYVRsA%2F5zwlVxHxn8rKDtLZSOyY%3D&shth=OSH.6eIp2z9SCp%252BJwPS6yUtJ%252Bw